README.rst

```diff
@@ -8,3 +8,121 @@ A GPL-licensed Python library that facilitates functional testing of quipucords.
    :target: https://travis-ci.org/quipucords/camayoc
 .. image:: https://codecov.io/gh/quipucords/camayoc/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/quipucords/camayoc
+
+Installation
+^^^^^^^^^^^^
+
+Camayoc only supports Python 3.5 and python 3.6, so it is recommended that you
+install Camayoc into a virtual environment. There are several tools available
+for managing virtual environments, below is an example of one way.::
+
+    git clone https://github.com/quipucords/camayoc.git
+    python3.6 -m vevn ~/envs/camayoc
+    source ~/envs/camayoc/bin/activate
+    cd camayoc
+    # To install for development, install all dependencies
+    make install-dev
+    # If you only want to run the test suite, you can skip
+    # a few dependencies with the plain "install" target
+    make install
+
+Configuration
+^^^^^^^^^^^^
+
+Camayoc requires a configuration file to be installed to
+``$XDG_CONFIG_HOME/camayoc/config.yaml``. On most systems this will be
+``~/.config/camayoc/config.yaml``.
+
+The config file tells camayoc about the hosts you would like to run tests
```

Comment:
Need consistency around `camayoc` or `Camayoc`. I'd uppercase it based on Jamie's previous opinions with Rho.